documentation/docs/demos/Cognite3DViewerDemo.tsx

```diff
@@ -43,6 +44,7 @@ export default function Cognite3DViewerDemo({
     addModel({ modelId, revisionId });
 
     window.viewer = viewer;
+    (window as any).DragControls = DragControls;
```

Comment:
Why not to put it into custom scope?

Reply:
Not sure how this ended up here as it's part of the clipping example, but how do I provide a custom scope @maksnester ?

Reply:
with the customScope file